src/context.rs

```diff
@@ -98,8 +98,8 @@ pub struct AllPreallocated<'buf> {
 mod private {
     use super::*;
     // A trick to prevent users from implementing a trait.
-    // on one hand this trait is public, on the other it's in a private module
-    // so it's not visible to anyone besides it's parent (the context module)
+    // On one hand this trait is public, on the other it's in a private module
+    // so it's not visible to anyone besides it's parent (the context module).
```

Comment:
In 5e07e7596b3c128c2290089f87a2851946fb538d:

Should also change the second `it's` to `its`

Reply:
Also we could probably drop the exposition here; it's a pretty standard idiom now.